docs-src/docs/setup/upgrade.md

```diff
@@ -3,18 +3,18 @@
 This section provides procedures for upgrading your Layer0 installation to the latest version.
 This assumes you are using Layer0 version `v0.10.0` or later. 
 
-!!! note 
+!!! warning 
     Layer0 does not support updating MAJOR or MINOR versions in place unless explicitly stated otherwise.
-    Users will need to destroy and re-create Layer0 instances in these circumstances. 
+    Users will either need to create a new Layer0 instance and migrate to it or destroy and re-create their Layer0 instance in these circumstances.
 
-Run the **upgrade** command, replacing `<instance_name>` and `<version>` with the name of the Layer0 instance and new version, respectively:
+Run the `upgrade` command, replacing `<instance_name>` and `<version>` with the name of the Layer0 instance and new version, respectively:
 ```
 $ l0-setup upgrade <instance_name> <version>
```

Comment:
Should we remove all '$' to have consistency

Reply:
Yes. I was aiming for that specific consistency.